<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>721</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_scene"/>
   </widget>
   <widget class="QPushButton" name="Lancer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>241</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Life</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="vie">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Food</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="miam">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="score_label">
       <property name="text">
        <string>Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="score"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_character">
       <property name="text">
        <string>Choose your character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="character">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Lancer</sender>
   <signal>clicked()</signal>
   <receiver>label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Lancer</sender>
   <signal>clicked()</signal>
   <receiver>vie</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Lancer</sender>
   <signal>clicked()</signal>
   <receiver>label_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Lancer</sender>
   <signal>clicked()</signal>
   <receiver>miam</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Lancer</sender>
   <signal>clicked()</signal>
   <receiver>Lancer</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Lancer</sender>
   <signal>clicked()</signal>
   <receiver>character</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
